static/locales/fr.client.json

```diff
@@ -739,7 +739,19 @@
     "menus": {
         "Select fields": "Sélectionner les champs",
         "* Workspaces are available on team plans. ": "* Les dossiers sont disponibles avec une offre équipe. ",
-        "Upgrade now": "Mettre à jour maintenant"
+        "Upgrade now": "Mettre à jour maintenant",
+        "Any": "Any",
```

Comment:
Is there a particular reason you've added these in `fr.client.json`? Did the code not work without them?